cmd/rookflex/cmd/root.go

```diff
@@ -34,13 +35,15 @@ const (
 	cephFS = "ceph"
 )
 
+// RootCmd the rookflex volume plugin cobra root command
 var RootCmd = &cobra.Command{
 	Use:           "rookflex",
 	Short:         "Rook Flex volume plugin",
 	SilenceErrors: true,
 	SilenceUsage:  true,
 }
 
+// Execute run the rookflex volume plugin RootCmd
 func Execute() {
```

Comment:
does this actually get called? `main()` calls RootCmd.Execute() directly, right?

Reply:
Yes, it seems that https://github.com/rook/rook/blob/ff136d627e84bce1391eabdc36143f1519d42a2b/cmd/rookflex/cmd/root.go#L44 is not called, I'll remove it and test.